### DISCUSSION TITLE
Avoid cash and transfers options

### DISCUSSION THREAD

**Discussion by EmmanuelSalazarID** [mercadopago/sdk-js#91] [Q&A] [answered]
Hi everyone, Is there any way to avoid or don't show cash and transfers options? and only accept credit and debit cards?

Now I'm using like this:
```
const mp = new MercadoPago("${KEY}", {
    locale: "es-MX",
  });
  mp.checkout({
    preference: {
      id: "${preferenceId}",
    },
    autoOpen: true
  });
  true
```
Is there any parameter or something to block that options (Cash and transfers)?

**Comment by lucmantovani** [comment, marked as answer]
Hello @EmmanuelSalazarID!

You can achieve this behavior by configuring the attribute `payment_methods. excluded_payment_types` when creating the preference. More details in the [documentation](https://www.mercadopago.com.mx/developers/es/reference/preferences/_checkout_preferences/post).
